src/com/yetanalytics/lrs_admin_ui/subs.cljs

```diff
@@ -69,6 +71,29 @@
  (fn [db _]
    (::db/new-account db)))
 
+(reg-sub
+ :db/update-password
+ (fn [db _]
+   (::db/update-password db)))
+
+(reg-sub
+ :update-password/old-password
+ :<- [:db/update-password]
+ (fn [update-password _]
+   (:old-password update-password)))
+
+(reg-sub
+ :update-password/new-password
+ :<- [:db/update-password]
+ (fn [update-password _]
+   (:new-password update-password)))
+
+(reg-sub
+ :update-password/valid?
+ :<- [:db/update-password]
+ (fn [update-password _]
+   (s/valid? ::i/valid-update-password update-password )))
```

Comment:
Extra space at the end

Reply:
https://github.com/yetanalytics/lrs-admin-ui/pull/43/commits/0ccd64bf4de54234056a9d6bff05ccc41f3584ac